scripts/Json-xlsx/README.md

```diff
@@ -0,0 +1,8 @@
+## Json to xlsx converter
+
+### Usage 
+1. Navigate to current dir using commandline 
+2. Run using -> python script.py
```

Comment:
Please add the command line arguments this script needs to run in the documentation.